addons/website/static/src/builder/plugins/image/image_transform_option.js

```diff
@@ -0,0 +1,26 @@
+import { BaseOptionComponent } from "@html_builder/core/utils";
+import { useTransformOperations } from "@html_editor/main/media/image_transform_button";
+import { registry } from "@web/core/registry";
+
+export class ImageTransformOption extends BaseOptionComponent {
+    static template = "website.ImageTransformOption";
+
+    setup() {
+        super.setup();
+        useTransformOperations({
+            document: this.env.editor.document,
```

Comment:
To be checked functionally with the PO's: Do we want the `ImageTransformation` to be removed when we click on the editor?